### DISCUSSION TITLE
Upgrading/Transferring a Project from Hobby to Pro

### DISCUSSION THREAD

**Discussion by ChrisDalley** [vercel/vercel#4164] [General] [answered]
Hi folks, 
I have come to redeploy a site and got hit with the limit to "12 Serverless functions" new limit. While I get the change and am completely happy with paying to upgrade to a pro account, it seems like the process for doing this is not documented anywhere. 

I have created a team but there is no way to move a project from my personal account to the team account. It seems like the only way to do this is to 
- create a brand new project
- remove zeit github app from personal account and reinstall under ream account
- read all the secrets all over again inside the team
- move the domain over from my personal account to the team
- deploy the new project
- repoint the domain at the new project rather than the old one
- reinstall any configured add-ons etc inside of zeit 
- delete the old project

This seems like a completely unecessary process just to get from 12 serverless functions to 24 and I'm just wondering if anyone knows of a better way of handling this (other than reducing the number of serverless functions) 

Any recommendations/help would be massively appreciated!

**Comment by coetry** [comment, marked as answer]
Hi @ChrisDalley we're well aware of this limitation and are working on making it easier to migrate projects across accounts! We know its a tedious paint point as of right now and really do apologize for the inconvenience!

**Reply by ChrisDalley** [reply]
@coetry - Thanks for the quick response here and through support. Appreciate the fast feedback, switching the project over turned to be less painful than expected (made alot easier with your ability to move domains across accounts)!

**Reply by william-bratches** [reply]
limitation? this is the core customer conversion process. hit the free tier, then pay. This abymsal UX is really disappointing. It's easier to just to throw this up on AWS.

**Comment by stewartmcgown** [comment]
Where we at on this? I'm about to upgrade to Pro but I am absolutely not gonna recreate like 12 projects in a seperate account.

**Reply by stewartmcgown** [reply]
I'm quite literally begging to give you money here. What can we do?

**Comment by IamManchanda** [comment]
+1 ... I need this feature too ...

**Reply by IamManchanda** [reply]
Ignore it ... I just found out that it's already possible  https://vercel.com/docs/platform/projects#transferring-a-project ... Big Cheers to vercel on amazing product :)
